packages/management/core-management-plugins/bundle/.imgpkg/images.yml

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
Is this file supposed to be committed?

Reply:
thanks for catching this, they shouldn't be commited